.env

```diff
@@ -0,0 +1,4 @@
+pg_user=identity
+pg_pass=identity
+pg_db_name=identity
+pg_port_host=5433
```

Comment:
5433? We should probably default to Postgres default port
(found in multiple files of the PR)

Reply:
Also, what is this file used for? The env names do not match the ones used by the server

Reply:
It's used by docker-compose